src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String segLine = "    ____________________________________________________________";
```

Comment:
```suggestion
    private static final String SEGLINE = "    ____________________________________________________________";
```
Constant names must be all uppercase using underscore to separate words.